DiscoveryInsightsinEinsteinAnalytics.apxc

```diff
@@ -0,0 +1,140 @@
+@RestResource (urlMapping = '/DiscoveryInsights' )
```

Comment:
Can you move this here `https://github.com/forcedotcom/AnalyticsApexSteps/tree/master/force-app/main/apex/examples/integration` and rename with `.cls` suffix.